### ISSUE TITLE
(Incompatibility issue) Show-me-your-skin, with - [Wildfire's Female Gender Mod 1.20.2-3.2]

### ISSUE THREAD

**Issue by Zorklis** [WildfireRomeo/WildfireFemaleGenderMod#163]
[Fabric] Wildfire's Female Gender Mod 1.20.2-3.1 ;
this previous version works fine but the current newer version (in the title) has some launch issues.
[crash-2023-12-04_08.08.31-client.txt](https://github.com/WildfireRomeo/WildfireFemaleGenderMod/files/13542480/crash-2023-12-04_08.08.31-client.txt)
says it's conflicting with "earlyloadingscreen" mod.
Removing that one gets the loading stuck at Mojang Studios, loading bar.
removing show-me-your-skin-1.7.0+1.20.2 fixes the conflict.
I put earlyloadingscreen mod back in and the game loaded into main menu. Plus that mod does it's job just fine.

TL:DR newest version has introduced incompatibility issue with [Show me Your Skin](https://github.com/enjarai/show-me-your-skin/releases/tag/1.7.0%2B1.20.2) mod.

**Comment by Zorklis** [comment]
[crash-2023-12-05_10.22.00-client.txt](https://github.com/WildfireRomeo/WildfireFemaleGenderMod/files/13557054/crash-2023-12-05_10.22.00-client.txt)
Crash log with the newer version (1.20.2-3.2.1), still crashing.
1.20.2-3.1 - is still good.

**Comment by WildfireRomeo** [comment]
This is related to how armor stands now render the custom armor features added in 3.2. It's not something I can fix, so I'm opening an issue for Show Me Your Skin.
